wiki/artifact-created-event-aggregation.md

```diff
@@ -1,5 +1,6 @@
-# ArtifactCreatedEvent Aggregation
+# ArtifactCreatedEvent aggregation
```

Comment:
In English the titles should be capitilized. https://www.webucator.com/how-to/how-capitalize-headings-titles.cfm

Reply:
Ok, I'll change back then :)